### PR TITLE
[Mellanox] Update SN2201 device files

### PR DESCRIPTION
**Pull request by keboliu**
Signed-off-by: Kebo Liu <kebol@nvidia.com>

#### Why I did it
update sensor conf, pcie.yaml and default sku for SN2201

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106

### CODE REVIEW

File: device/mellanox/x86_64-nvidia_sn2201-r0/sensors.conf [hunk 1]
```diff
@@ -33,15 +90,71 @@ bus "i2c-8" "i2c-1-mux (chan_id 6)"
...
 bus "i2c-9" "i2c-1-mux (chan_id 7)"
     chip "pmbus-i2c-*-40"
-    ignore curr3
-    ignore in3
-    ignore power3
+        label curr1 "VR IC PSU 12V Rail curr(in)"
+        label curr2 "ASIC 0.9V VCORE Rail Curr(out)"
+        label in1 "VR IC PSU 12V Rail"
+        label in2 "VR IC ASIC 0.9V VCORE Rail"
+        label power1 "VR IC PSU 12V Rail pwr(in)"
```
Comment (Junchao-Mellanox) [inline device/mellanox/x86_64-nvidia_sn2201-r0/sensors.conf]: pwr -> Pwr?

File: device/mellanox/x86_64-nvidia_sn2201-r0/sensors.conf [hunk 2]
```diff
@@ -33,15 +90,71 @@ bus "i2c-8" "i2c-1-mux (chan_id 6)"
 # Power controllers
 bus "i2c-9" "i2c-1-mux (chan_id 7)"
     chip "pmbus-i2c-*-40"
-    ignore curr3
-    ignore in3
-    ignore power3
+        label curr1 "VR IC PSU 12V Rail curr(in)"
```
Comment (Junchao-Mellanox) [inline device/mellanox/x86_64-nvidia_sn2201-r0/sensors.conf]: curr -> Curr?

File: device/mellanox/x86_64-nvidia_sn2201-r0/sensors.conf [hunk 3]
```diff
@@ -33,15 +90,71 @@ bus "i2c-8" "i2c-1-mux (chan_id 6)"
...
+        label in9 "MONITOR CPU Board V1P24"
+
+# PSU PMBus sensors
+bus "i2c-3" "i2c-1-mux (chan_id 1)"
+    chip "pmbus-i2c-3-58"
+        label in1 "PSU-1 220V Rail(in)"
+        label in2 "PSU-1 12V Rail(out)"
+        ignore in3
+        label fan1 "PSU-1 fan1"
+        label temp1 "PSU-1 temp1"
```
Comment (Junchao-Mellanox) [inline device/mellanox/x86_64-nvidia_sn2201-r0/sensors.conf]: temp1 -> Temp1? And same for other temperature label.

File: device/mellanox/x86_64-nvidia_sn2201-r0/sensors.conf (hunk 1 above)
Comment (keboliu) [reply device/mellanox/x86_64-nvidia_sn2201-r0/sensors.conf]: fixed

File: device/mellanox/x86_64-nvidia_sn2201-r0/sensors.conf (hunk 2 above)
Comment (keboliu) [reply device/mellanox/x86_64-nvidia_sn2201-r0/sensors.conf]: fixed

File: device/mellanox/x86_64-nvidia_sn2201-r0/sensors.conf (hunk 3 above)
Comment (keboliu) [reply device/mellanox/x86_64-nvidia_sn2201-r0/sensors.conf]: fixed

**Review by Junchao-Mellanox — APPROVED** [review]

**Pull request closed without merging**